static/src/javascripts/projects/common/views/experiments/remind-me-email-signup.html

```diff
@@ -0,0 +1,11 @@
+<div class="remind-me">
+    <% if (subtitle) { %>
+    <h2 class="remind-me__subtitle"><%= subtitle %></h2>
+    <% } %>
+    <h1 class="remind-me__title"><%= title %></h1>
```

Comment:
Yep fair point! I guess because it's the main title of this component but of course there are more important headings elsewhere in the page - kind of forgot I wasn't inside an iframe when writing that (did a lot of experiments with embeds previously). Fixed